recipes/cget/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cget" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2a7913b601bec615208585eda7e69998a43cc17080d36c2ff2ce8742c9794bf6
+
+build:
+  number: 0
+  entry_points:
+    - cget = cget.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - click >=6.6
+    - six >=1.10
+    - subprocess32  # [py27]
+  run:
+    - python
+    - click >=6.6
+    - six >=1.10
+    - cmake
+    - subprocess32  # [py27]
```

Comment:
Conda-forge doesn't support Python 2 any more so this can be remove so the recipe can be `noarch: python`:
```suggestion
```

Reply:
@chrisburr I realize that, but I am considering an admin command for adding it, at least to provide a migration option for scientific scripts depending on this package and python 2.7. What do you think is better, to make this PR only python3 and then branch out with python2? [2020-03-21: Python 2.7 Admin Command Available](https://conda-forge.org/docs/user/announcements.html) or would be possible to do python 2.7 with build # 0 and then head for python 3 making another build (# 1)?

Reply:
In that case I would go for depending on `- subprocess32` for all Python versions as I recently added a Python 3 build of it: https://github.com/conda-forge/subprocess32-feedstock/pull/15

It'll be easier than using the bot command (which didn't work last time I tried and I had to do it by hand).

Reply:
@chrisburr thanks for the tips. In that case does it make sense to have subprocess32 in host section too and only removing others? Or do you think that removing it wouldn't have effect when for example cget is being a build dependency used in other package? (obviously cget doesn't run without)

Reply:
Python packages should normally only need `python` and `pip`. The `host` section referrs to what is required to "build" `cget` and has effect impact once the package is built.

Reply:
@chrisburr ok, thanks! updating..